resources/views/components/table.blade.php

```diff
@@ -14,7 +14,7 @@
     <tr>
       {{-- Renders all the headers --}}
       @foreach ($headers as $header)
-        <th class="px-3 py-3">
+        <th class="px-3 py-3" width="{{ is_object($header) ? $header->width : '' }}">
```

Comment:
Should we validate if `width` has a value?  or to set an empty string as a default value of this property? what do you think?

Reply:
I think that is a good idea